lib/pluginmanager/bundler/logstash_uninstall.rb

```diff
@@ -93,10 +97,19 @@ def unfreeze_gemfile
       end
     end
 
-    def self.uninstall!(gem_name, options = { :gemfile => LogStash::Environment::GEMFILE, :lockfile => LogStash::Environment::LOCKFILE })
-      gemfile_path = options[:gemfile]
-      lockfile_path = options[:lockfile]
-      LogstashUninstall.new(gemfile_path, lockfile_path).uninstall!(gem_name)
+    def mutate_gemfile
```

Comment:
Name suggestion (take it or leave it) `with_writable_gemfile`. Method names that start with `with` help indicate its a context manager (though i realize that `unfreeze_gemfile` does not really follow this pattern anyway).